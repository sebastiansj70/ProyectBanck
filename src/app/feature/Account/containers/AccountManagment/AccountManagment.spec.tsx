import '@testing-library/jest-dom';
import { RenderResult, render } from '@testing-library/react';
import { AccountManagment } from './';
import React from 'react';

describe('My account', () => {
  let componenWrapper: RenderResult;
  let componentProps: React.ComponentProps<typeof AccountManagment>;

  beforeEach(() => {
    componentProps = {
      accountLists: {
        cedula: '123456789',
        nombres: 'Juan Sebastian',
        apellidos: 'Sanchez Parada',
        cuentas: [],
      },
      accountList: jest.fn(),
      saveAccount: jest.fn(),
      status:'ok'
    };

    componenWrapper = render(<AccountManagment {...componentProps} />);
  });

  it('should match snapshot', () => {
    expect(componenWrapper.container).toMatchSnapshot();
  });

  it('contain my account', () => {
    const element = componenWrapper.container;
    const tittle = element.querySelectorAll('h2');

    expect(tittle[0].textContent).toMatch(/Cuentas/);
  });
});
