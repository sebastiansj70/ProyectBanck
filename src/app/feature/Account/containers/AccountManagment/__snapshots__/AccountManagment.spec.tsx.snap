// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`My account should match snapshot 1`] = `
<div>
  <div
    class="account"
  >
    <h2
      class="tittle-account"
    >
      Mis Cuentas
    </h2>
    <div
      class="body-account"
    />
  </div>
</div>
`;
